nixos/tests/mpv.nix

```
@@ -12,7 +12,8 @@ in
     {
       environment.systemPackages = [
         pkgs.curl
-        (pkgs.wrapMpv pkgs.mpv-unwrapped {
+        (pkgs.mpv-unwrapped.wrapper {
+          mpv = pkgs.mpv-unwrapped;
```

I think this could be:
```suggestion
        (pkgs.mpv.override {
```